shared-libs/server-checks/src/checks.js

```diff
@@ -63,7 +63,7 @@ const couchDbVersionCheck = (serverUrl) => {
         return reject(err);
       }
       console.log(`CouchDB Version: ${body.version}`);
-      resolve();
+      resolve(body.version);
```

Comment:
This means every time the code calls `getCouchDbVersion` we'll get the version output in the log. While you're currently only calling it once per start I can see this function being used more. Can we split it so there's an exported function that just returns the version, and a "private" function that does the logging?

While we're here, shouldn't `couchDbVersionCheck` throw an error if the version is < 2.0? Maybe that's a separate issue...

Reply:
I pulled the function out separately. 
I'm not sure what should be done about the actual version "check". The shared-lib contained an exact copy of the code that existed in API's `server.js`. I think it should be a separate issue, at least to describe which version we're supporting at minimum. Maybe we will want to minimally support `2.3.1` (after release)?